sub-packages/bionemo-geneformer/tests/bionemo/geneformer/test_stop_and_go.py

```diff
@@ -46,7 +46,7 @@
 from bionemo.testing.harnesses.mode import Mode
 
 
-DATA_PATH: pathlib.Path = load("single_cell/testdata-20240506") / "cellxgene_2023-12-15_small" / "processed_data"
+DATA_PATH: pathlib.Path = load("single_cell/testdata-memmap-format", source="pbss") / "cellxgene_2023-12-15_small_mmap"
```

Comment:
Can you remove all of the `source="pbss"` from your tests etc. The default when you run can be changed with an environment variable. For example if you want to test locally and you want the test to use pbss then run:

```
export BIONEMO_DATA_SOURCE="pbss"
py.test [your cmd here]
```

Reply:
We do this in CI as well. Once you know stuff works and we like the files then you should work with @dorotat-nv to get them into NGC so the demo notebook etc still runs externally on brev.dev.